### ISSUE TITLE
Time related Question

### ISSUE THREAD

**Issue by nsankar** [NetManAIOps/CIRCA#2]
@limjcst  @lizeyan  Thanks for developing this interesting approach and the package. .  I have the following question

1. In the below example code, there are 6 sample data for latency, traffic and saturation. What is the sampling time considered per data sample? Is it  60 seconds (1 minute) for each data sample ?
2. Detect time mentioned is :  detect_time=240, Does this mean that in the 4th sample in the example data  viz., latency =9, traffic=105, and saturation =6  corresponds to the fault data i.e. (60*4 = 240)**?**
3. What do the lookup_window and the detect_window settings signify?

Kindly explain. Thanks in advance.

`# 2. Prepare data
mock_data = {
    latency: (10, 12, 11, 9, 100, 90),
    traffic: (100, 110, 90, 105, 200, 150),
    saturation: (5, 4, 5, 6, 90, 85),
}
mock_data_with_time: Dict[str, Dict[str, Sequence[Tuple[float, float]]]] = defaultdict(
    dict
)
for node, values in mock_data.items():
    mock_data_with_time[node.entity][node.metric] = [
        (index * 60, value) for index, value in enumerate(values)
    ]
data = CaseData(
    # circa.model.data_loader.MemoryDataLoader is derived from
    # circa.model.data_loader.DataLoader, which manages data with configurations
    data_loader=MemoryDataLoader(mock_data_with_time),
    sli=latency,
    detect_time=240,
    lookup_window=4,
    detect_window=2,
)

# 3. Conduct root cause analysis one minute after a fault is detected
print(model.analyze(data=data, current=data.detect_time + 60))`

**Comment by nsankar** [comment]
@limjcst Thank you for clarifying.

When I ran the toy example , I got the following output. How to interpret the results? Is it correct to say that the metric with the highest score is the root cause? For instance here, score for Saturation is the highest i.e. 383.251. So saturation caused is the root cause of the problem in question. correct?

`root@9d6d53736e6c:/app# python3 testRca.py
[(Node('DB', 'Saturation'), {'score': 383.2518754031084, 'info': {'z-score': 383.2518754031084, 'Confidence': 1.0}}), (Node('DB', 'Latency'), {'score': 355.00000000000006, 'info': {'z-score': 355.00000000000006, 'Confidence': 1.0}}), (Node('DB', 'Traffic'), {'score': 12.24744871391589, 'info': {'z-score': 12.24744871391589, 'Confidence': 1.0}})]`

**Comment by nsankar** [comment]
@limjcst  Appreciate your guidance.
